src/views/components-demo/count-to.vue

```diff
@@ -0,0 +1,276 @@
+        target="_blank"
+      >countTo-component</a>
+    </aside>
+    <count-to
+      ref="count"
+      :start-val="_startVal"
+      :end-val="_endVal"
+      :duration="_duration"
+      :decimals="_decimals"
+      :separator="_separator"
+      :prefix="_prefix"
+      :suffix="_suffix"
+      :autoplay="false"
+      class="count"
+    />
+    <div style="margin-left: 25%;margin-top: 40px;">
+      <label
+        class="label"
+        for="startValInput"
+      >
+        startVal:
+        <input
+          v-model.number="setStartVal"
+          type="number"
+          name="startValInput"
+        >
+      </label>
+      <label
+        class="label"
+        for="endValInput"
+      >
+        endVal:
+        <input
+          v-model.number="setEndVal"
+          type="number"
+          name="endVaInput"
+        >
+      </label>
+      <label
+        class="label"
+        for="durationInput"
+      >
+        duration:
+        <input
+          v-model.number="setDuration"
+          type="number"
+          name="durationInput"
+        >
+      </label>
+      <div
+        class="startBtn count-btn"
+        @click="start"
+      >
+        Start
+      </div>
+      <div
+        class="pause-resume-btn count-btn"
+        @click="pauseResume"
+      >
+        pause/resume
+      </div>
+      <br>
+      <label
+        class="label"
+        for="decimalsInput"
+      >
+        decimals:
+        <input
+          v-model.number="setDecimals"
+          type="number"
+          name="decimalsInput"
+        >
+      </label>
+      <label
+        class="label"
+        for="separatorInput"
+      >
+        separator:
+        <input
+          v-model="setSeparator"
+          name="separatorInput"
+        >
+      </label>
+      <label
+        class="label"
+        for="prefixInput"
+      >
+        prefix:
+        <input
+          v-model="setPrefix"
+          name="prefixInput"
+        >
+      </label>
+      <label
+        class="label"
+        for="suffixInput"
+      >
+        suffix:
+        <input
+          v-model="setSuffix"
+          name="suffixInput"
+        >
+      </label>
+    </div>
+    <aside>
+      &lt;count-to :start-val=&#x27;{{ _startVal }}&#x27; :end-val=&#x27;{{ _endVal }}&#x27; :duration=&#x27;{{ _duration }}&#x27;
+      :decimals=&#x27;{{ _decimals }}&#x27; :separator=&#x27;{{ _separator }}&#x27; :prefix=&#x27;{{ _prefix }}&#x27; :suffix=&#x27;{{ _suffix }}&#x27;
+      :autoplay=false&gt;
+    </aside>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import countTo from 'vue-count-to'
+
+@Component({
+  components: {
+    countTo
+  }
+})
+export default class CountToDemo extends Vue {
+  private setStartVal = 0
+  private setEndVal = 2017
+  private setDuration = 4000
+  private setDecimals = 0
+  private setSeparator = ''
+  private setSuffix = ' rmb'
+  private setPrefix = '¥ '
+
+  get _startVal() {
```

Comment:
can we use camelcase? (remove the '_' prefix)

Reply:
already edited